tests/stress/utils.py

```diff
@@ -22,7 +22,11 @@ def check_queue_status(duthost, queue):
     bgp_neighbors = duthost.show_and_parse(SHOW_BGP_SUMMARY_CMD)
```

Comment:
Could you please help to check whether there are other scripts that use the output of this cmd? thanks

Reply:
Updated one more place, this should be all the place related to the typo